src/game.cpp

```diff
@@ -2712,17 +2717,20 @@ void Game::toggleFreeMoveAlt()
 
 void Game::toggleFast()
 {
-	static const wchar_t *msg[] = { L"fast_move disabled", L"fast_move enabled" };
 	bool fast_move = !g_settings->getBool("fast_move");
 	g_settings->set("fast_move", bool_to_cstr(fast_move));
 
 	runData.statustext_time = 0;
-	m_statustext = msg[fast_move];
-
-	bool has_fast_privs = client->checkPrivilege("fast");
 
-	if (fast_move && !has_fast_privs)
-		m_statustext += L" (note: no 'fast' privilege)";
+	if (fast_move)
+	{
+		if (client->checkPrivilege("fast"))
+			m_statustext = narrow_to_wide(gettext("Fast mode enabled"));
+		else
+			m_statustext = narrow_to_wide(gettext("Fast mode enabled (note: no 'fast' privilege)"));
```

Comment:
Not really a big deal IMO, we are only talking about 3 strings here.
A fix would probably involve string concatenation, which is a definite no-go.